spec/features/user_submissions_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+feature "zgloszenia" do
+
+  context "po zalogowaniu" do
+    include_context "user login"
+    
+    before :each do
+        Submission.create!(person: Person.first, status: "odrzucony", polish_title: "Alicja w krainie czarów",
+                             english_title: "Alice in Wonderland", english_abstract: "Little about that story",
+                             english_keywords: "alice", received: "19-01-2016", language: "polski", issue: Issue.first)
```

Comment:
Tutaj brakuje danych:
* redaktora zgłoszenia - to jest atrybut `Person`
* autora
Żeby to poprawić - przed Submission proszę dodać:
```ruby
Person.create!(name: "Andrzej", surname: "Kapusta", email: "a.kapusa@gmail.com", sex:
                       "mężczyzna", roles: ['redaktor'], discipline:["psychologia"])
Person.create!(name: "Anna", surname: "Genialna", email: "user@localhost.com", sex:
                       "kobieta", roles: ['autor'], discipline:["psychologia"])
```
a po Submission:
```ruby
Authorship.create!(person: Person.last, submission: Submission.first, corresponding: true, position: 1)
```